examples/next-js-simple/README.md

```diff
@@ -1,45 +1,147 @@
+    <figcaption style="text-align: center">Enter a name for your new space, such as "My Next.js App".</figcaption>
+</figure>
+
+Now that you have a new space, the next step is connecting "My Next.js App" with your application.
+
+### Connecting Builder.io to your application
+
+To connect your Builder.io space and your application, set the site URL and get the API key as follows:
+
+1. In Builder.io, click on the Account icon on the left sidenav.
+
+    <figure>
+       <img src="https://cdn.builder.io/api/v1/image/assets%2FYJIGb4i01jvw0SRdL5Bt%2Fd26eb45cd52a4cada42413cd5c99419d" alt="Account icon in left sidenav" width="200" style="display: block; margin: auto"/>
+       <figcaption style="text-align: center">The Account icon takes you to important data about this space.</figcaption>
+    </figure>
+
+1. Change the Site URL to `http://localhost:3000` and click to copy the Public API Key.
+
+    <figure>
+       <img src="https://cdn.builder.io/api/v1/image/assets%2FYJIGb4i01jvw0SRdL5Bt%2Faec18bf8ccf543ab94c4230a7791c894" alt="Change the Site URL in account settings." width="400" style="display: block; margin: auto"/>
+       <figcaption style="text-align: center">The Site URL and API Key are in the **Space** tab.</figcaption>
+    </figure>
+
+1. In your code editor, configure `.env.production` and `.env.development` with the [Public API Key](https://builder.io/account/space) by adding a line to each file as follows, but using your Public API Key that you copied in the previous step. 
+    For example:
+
+    ```shell
+    BUILDER_PUBLIC_KEY=08837cee608a405c806a3bed69acfe2d <-- replace this with your API Key
+    ```
+
+## Running your application
+
+To serve your application locally, install dependencies, serve, and view your preview. 
+
+1. Install dependencies by entering the follw\owing at the command line. 
+
+    ```
+    npm install
+    ```
+
+1. Serve your application by running the following at the command line:
+
+    ```
+    npm run dev
+    ```
+
+1. In your browser, go to `http://localhost:3000` to see your application.
+
+### Experimenting
+
+Now that you have a configured Builder.io application, you can try different features, such as creating a page.
+Create a new page entry, assign any URL, publish and preview.
+For more detail and ideas on creating pages, see [Creating a landing page in Builder
+](https://www.builder.io/c/docs/creating-a-landing-page).
 
 ### Deploy
 
 You can deploy anywhere you like, but for this project we recommend [Vercel](https://nextjs.org/docs/deployment).
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fbuilderio%2Fbuilder%2Ftree%2Fmaster%2Fexamples%2Fnext-js-simple)
+
+
+## Next steps
+
+* For more information on previewing your applications, see [Editing and previewing directly on your site](https://www.builder.io/c/docs/guides/preview-url).
```

Comment:
This readme is incredible. Only suggestion is lets point people how to register their react components too. Please feel free to change copy

```suggestion
* Learn how to [use your react components in our visual editor](https://www.builder.io/c/docs/custom-react-components)
* For more information on previewing your applications, see [Editing and previewing directly on your site](https://www.builder.io/c/docs/guides/preview-url).
```

Reply:
Meant to note this upon submission: resolved in https://github.com/BuilderIO/builder/pull/558